search/elastic.py

```diff
@@ -45,6 +46,11 @@ def translate_result(result):
     return response
 
 
+def _is_iterable(item):
```

Comment:
Why are we coping the same function twice? Maybe you should move this over to helpers.